src/components/Spinner/Spinner.tsx

```diff
@@ -14,11 +14,16 @@ type SpinnerStroke = "thin" | "medium" | "bold";
 type Intent = "inherit" | "primary" | "success" | "warning" | "danger";
```

Comment:
Without allowing "none" (an original value from BlueprintJS, and also GUI elements has it in the shared `Intent` type) the Explore app won't transpile. `none` has been allowed because `intent` was just forwarded so far.